Public/Src/Cache/ContentStore/Library/Stores/FileSystemContentStoreInternalChecker.cs

```diff
@@ -201,17 +200,15 @@ private void RemoveSelfCheckStateFromDisk(Context context)
             else
             {
                 // The loop was interrupted. Saving an incremental state.
-                var newStatus = selfCheckState.WithEpochAndPosition(_settings.SelfCheckEpoch, contentHashes[index].hash);
+                var newStatus = selfCheckState.WithEpochAndPosition(_settings.SelfCheckEpoch, contentHashes[index].Hash);
                 UpdateSelfCheckStateOnDisk(context, newStatus);
             }
 
             return Result.Success(new SelfCheckResult(invalidHashes: invalidEntries, totalProcessedFiles: processedEntries));
 
-            int findNextIndexToProcess(ContentHash lastProcessedHash)
+            int FindNextIndexToProcess(ContentHash lastProcessedHash)

```

Comment:
we use camelCase naming convention for local functions. So I think we should rename it back.